submissions/vovan-zt/DOM/script.js

```diff
@@ -0,0 +1,102 @@
+window.addEventListener ('DOMContentLoaded', () => {
+"use strict";
```

Comment:
Please add "use strict" at the top of your scripts, otherwise, the strict mode may not be enabled.
In the loading function set only the initial function calls and event listeners.